src/casing.f90

```diff
@@ -135,9 +102,6 @@ subroutine casing( teta, zeta, gBn, icasing )
 
   BEGIN(casing)
 
-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!-!
-
-  jk = globaljk ! shorthand; globaljk is a "global" variable which must be passed through to subroutine dvcfield;
```

Comment:
I don't understand (yet) why this can be left out. `jk` is still present as a local variable in line 96, hence I assume that it is still used. Isn't this the way how `bnorml` communicates to `casing` for which point on the computation boundary to compute the virtual casing integral? Please fill me in on the thinking that went into this :)

Reply:
globaljk for dvcfield must be set one level higher than this, in the bnorml routine, so I thought this is dead code, but as you correctly pointed out jk is still used for printing. Reverted.